app/views/layouts/_application_content.html.erb

```diff
@@ -2,19 +2,42 @@
   <div id="content" class="container">
     <%= notifications %>
     <% unless @page_header.blank? %>
-      <div class="row form-group">
-        <h1 class="col-md-8"><%=h @page_header %></h1>
-      </div>
+        <div class="row form-group">
+          <h1 class="col-md-8"><%=h @page_header %></h1>
+        </div>
+    <% end %>
+    <% if @count_nil_hosts != nil && @count_nil_hosts > 0 %>
+        <%= alert :class => 'alert-warning', :header => '',
+                  :text => (n_('There is', 'There are', @count_nil_hosts) + ' ' +
+                      link_to(n_("%{count} host with no %{taxonomy_single} assigned",
+                                 "%{count} hosts with no %{taxonomy_single} assigned", @count_nil_hosts) %
+                                  {:count => @count_nil_hosts , :taxonomy_single => taxonomy_single },
+                              hosts_path(:search => "not has #{controller_name.singularize}"))).html_safe %>
+    <% end %>
+    <% if searchable? && !(@page_header == "Audits" || @page_header == "Overview")  %>
```

Comment:
any suggestion here for a better way to catch this pages? what if we have/ will have more pages except of this two?